src/patterns/dosdp-patterns/entity_attribute_partOf.yaml

```diff
@@ -0,0 +1,68 @@
+pattern_name: entity_attribute
```

Comment:
The pattern_name does not look right.
The partial camel case in the file name looks inconsistent with other pattern yaml filenames.